cpp/oneapi/dal/algo/decision_forest/backend/cpu/train_kernel_regression_dense.cpp

```diff
@@ -132,6 +132,7 @@ static train_result<Task> call_daal_kernel(const context_cpu& ctx,
         daal::services::internal::hostApp(daal_input),
         daal_data.get(),
         daal_labels.get(),
+        nullptr, // no weights
```

Comment:
Please add full weights setting/getting support for train input, thus it can be used here instead of nullptr.